README.md

```diff
@@ -1 +1,118 @@
-# issac_ros2_control
+# Isaac Ros2 Control
+
+Description: This is a ROS 2 package for integrating the ros2_control with the [NVIDIA Isaac Sim](https://developer.nvidia.com/isaac-sim).
+
+
+<img src="https://picknik.ai/assets/images/logo.jpg" width="100">
+
+[![Build Status](https://github.com/PickNikRobotics/isaac_ros2_control/actions/workflows/build_and_test.yaml/badge.svg)](https://github.com/PickNikRobotics/isaac_ros2_control/actions/workflows/build_and_test.yaml)
+
+## Installation
+
+### Build from Source
+
+These instructions assume you are running on Ubuntu 20.04:
+
+1. [Install ROS2 humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html). You can stop following along with the tutorial after you complete the section titled: [Environment setup](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html#environment-setup). Make sure you setup your environment with:
+
+        source /opt/ros/humble/setup.bash
+
+   > *NOTE:* You may want to add that line to your `~/.bashrc`
+
+2. [Install ROS2 Build Tools](https://docs.ros.org/en/humble/Installation/Ubuntu-Development-Setup.html#install-development-tools-and-ros-tools)
+
+   > *NOTE:* If installing on a fresh OS, run `sudo rosdep init` and `rosdep update` before the install script
+
+3. Create a colcon workspace (*Note:* Feel free to change `~/ws_ros2` to whatever absolute path you want):
+
+        export COLCON_WS=~/ws_ros2/
+        mkdir -p $COLCON_WS/src
+
+4. Get the repo:
+
+        cd $COLCON_WS/src
+        git clone git@github.com:PickNikRobotics/isaac_ros2_control.git
+
+5. Download the required repositories and install any dependencies:
+
+        cd $COLCON_WS/src
+        vcs import < isaac_ros2_control/isaac_ros2_control.repos
+        rosdep install --ignore-src --from-paths . -y
+
+        # Pick a ROS_DOMAIN_ID that doesn't clash with others
+        echo 'export ROS_DOMAIN_ID='<YOUR-NUMBER> >> ~/.bashrc
+
+7. Configure and build the workspace:
+
+        cd $COLCON_WS
+        colcon build --symlink-install --event-handlers log-
+
+8. Source the workspace.
+
+        source $COLCON_WS/install/setup.bash
+
+> *Note*: Whenever you open a new terminal be sure to run `source /opt/ros/humble/setup.bash`, `export COLCON_WS=~/ws_ros2`, and `source $COLCON_WS/install/setup.bash`. You may want to add those commands to your `~/.bashrc`
+
+## For Developers
```

Comment:
Need to add a note that Isaac defaults its cycle time to ~60Hz and that the ros2_control yaml should match otherwise it will waste resources if run faster or have issues if run slower.